.gitattributes

```diff
@@ -1,3 +1,8 @@
+# Linux scripts - they need to have Linux line endings or Docker barfs
```

Comment:
Something more explanatory than "barfs" would be appropriate here.

Reply:
Yeah - that'll be in the book